docs/service-normalizationservice.html

```
@@ -406,6 +406,16 @@ <h3 id="supported-operations">Supported Operations <a id="supported-operations"
 Modifies <code>Period.start</code> to match the precision of <code>Period.end</code>.
 Prevents errors in the CQL (measure evaluation) engine caused by inconsistent precision in Period types.
 </Accordion>
+<Accordion title="Remove Extensions">
+Removes top-level FHIR extensions from a resource based on a configurable list of extension URLs.</p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)